src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="INFO">
+    <Appenders>
+        <Console name="LogToConsole" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+        </Console>
+        <File name="LogToConsole" fileName="logs/app.log">
+            <PatternLayot>
+                <Pattern>%d %c %m%n</Pattern>
+            </PatternLayot>
+        </File>
+    </Appenders>
+    <Loggers>
+        <Logger name ="mate.academy" level="info" additivity="false">
+            <AppenderRef ref="LogToFile"/>
```

Comment:
there is no LogToFile appender in your file, but are 2 LogToConsole

Reply:
this is my mistake, I wrote LogToConsole instead of LogtoFile